src/Template/Element/Flash/default.ctp

```diff
@@ -0,0 +1,11 @@
+<?php
+$class = 'alert';
+if (!empty($params['class'])) {
```

Comment:
picky point, would be "nicer" I think if class, if it's present, is expected to be an array.

i.e.:

    if (!empty($params['class'])) {
        $classes = (array)$params['class'];
    } else {
        $classes[] = 'alert-info';
    }
    $classes[] = 'alert';
    $classes = array_unique($classes);
    ?>
    <div class="<?= h(implode(' ', $classes)); ?>">
    
Makes conditional logic slightly easier, such as:

    <div class="<?= h(implode(' ', $classes)); ?>">
    <?php if (in_array('alert-dismissable', $classes)): ?>
        <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
    <?php endif; ?>